src/test/java/com/thindeck/dynamo/DyReposTest.java

```diff
@@ -128,6 +130,31 @@ public void iteratesOverMultipleRepos() throws IOException {
     }
 
     /**
+     * DyRepos can return the default user if the user was not specified.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void returnsDefaultUser() throws Exception {
+        MatcherAssert.assertThat(
+            new DyRepos(Mockito.mock(Region.class)).user(),
+            Matchers.is(User.DEFAULT)
+        );
+    }
+
+    /**
+     * DyRepos can return the specified user if the user was not specified.
```

Comment:
The comment is not correct, it should be *DyRepos can return the specified user.*, without the rest.